doc/website/siteConfig.js

```diff
@@ -0,0 +1,43 @@
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+
+const siteConfig = {
+  title: 'Coursier',
+  tagline: 'Pure Scala Artifact Fetching',
+
+  // wiped when relativizing stuff
+  url: 'https://coursier.github.io',
+  baseUrl: '/coursier/',
+
+  projectName: 'coursier',
+  organizationName: 'coursier',
+
+  customDocsPath: 'processed-docs',
+
+  headerLinks: [
+    {doc: 'intro', label: 'Docs'},
+    {href: 'https://github.com/coursier/coursier', label: 'GitHub'},
+  ],
+
+  users: [],
+
+  colors: {
+    primaryColor: '#33475B',
+    secondaryColor: '#3498DB',
+  },
+
+  copyright: `Copyright © ${new Date().getFullYear()} Alexandre Archambault and coursier contributors`,
+
+  highlight: {
+    theme: 'default',
+  },
+
+  scripts: ['https://buttons.github.io/buttons.js'],
+
+  onPageNav: 'separate',
+  cleanUrl: true,
+
+  enableUpdateTime: true, // doesn't seem to work
```

Comment:
I also had problems with this option, I think it was a recent contribution

Reply:
They seem to [use it](https://github.com/facebook/Docusaurus/blob/7d4d9fe96127c96aa77a1d57a138174ed617db7c/v1/website/siteConfig.js#L72) successuflly on their own website though.

Reply:
Seems that's because they expect the `.md` files to be committed, see [here](https://github.com/facebook/Docusaurus/blob/0f8535293eefddd41dc59e6ad622f6fcac1ec7b8/v1/lib/core/DocsLayout.js#L50).

Reply:
Makes sense, although it might be nice to fall-back to the file timestamp.

Reply:
Agreed...